src/election_works/entities/election.clj

```
@@ -577,17 +577,19 @@
          set)))
 
 (def create-schema
-  {(s/optional-key :id) s/Uuid
-   :description s/Str
-   :date s/Inst
-   :district-divisions #{election-schema/election-district-division-schema}
-   (s/optional-key :google-election-id) s/Str
-   (s/optional-key :population) (s/pred integer?)
-   (s/optional-key :website) s/Str
-   (s/optional-key :polling-place-url) s/Str
-   (s/optional-key :polling-place-url-shortened) s/Str
-   (s/optional-key :local-voting-methods) #{election-schema/local-voting-method-schema}
-   (s/optional-key :source) election-schema/source-schema})
+  (s/constrained
+   {(s/optional-key :id) s/Uuid
+    :description s/Str
+    :date s/Inst
+    :district-divisions #{election-schema/election-district-division-schema}
+    (s/optional-key :google-election-id) s/Str
+    (s/optional-key :population) (s/pred integer?)
+    (s/optional-key :website) s/Str
+    (s/optional-key :polling-place-url) s/Str
+    (s/optional-key :polling-place-url-shortened) s/Str
+    (s/optional-key :local-voting-methods) #{election-schema/local-voting-method-schema}
+    (s/optional-key :source) election-schema/source-schema}
+   election-schema/check-order!))
```

I think your error message gets swallowed by schema if it's added as a constraint -- we might need to check it separately.